esp-hal/src/macros.rs

```diff
@@ -26,6 +26,23 @@ macro_rules! before_snippet {
     };
 }
 
+#[doc(hidden)]
+#[macro_export]
+macro_rules! trm_markdown_link {
```

Comment:
This is a great idea!

Could you add a small comment on this macro showing how to obtain the anchor? At least the PDF viewer in Chrome only seems to be able to give me a page number link `#page=blah`, which is fine, but the section anchors would be better in the case where the TRM is updated.

Reply:
Ah I just read that @bjoernQ mentioned the anchor thing, how did you find the `#iomux` value?

Reply:
From esp-idf documentation. https://docs.espressif.com/projects/esp-idf/en/stable/esp32/api-reference/peripherals/gpio.html has a [PDF] link.

Reply:
> Could you add a small comment on this macro showing how to obtain the anchor?

I have absolutely no idea 🤣

Reply:
Let's figure this out another time